docs/running-on-kubernetes.md

```diff
@@ -816,6 +816,15 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
   <td>2.3.0</td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.driver.service.annotation.[AnnotationName]</code></td>
+  <td>(none)</td>
+  <td>
+    Add the annotation specified by <code>AnnotationName</code> to the driver service.
```

Comment:
Nit: may worth mentioning "Kubernetes annotations" and adding a link to the Kubernetes doc on annotations.

Reply:
Thank you! Sure!